### PR TITLE
Testing the todolist

### PR DESCRIPTION
**Pull request by saimaAnis21**
- To test the different modules in todolist project

### CODE REVIEW

**Review by NtwaliHeritier — CHANGES_REQUESTED** [review]
### **STATUS: CHANGES REQUIRED🛑🛑🛑**

Hello @saimaAnis21 ,
Great work on this project. I can tell you put in a lot of effort👏👏
Let's make few improvements and get it approved

[REQUIRED]: Increase the number of test cases. You can do this by adding negative test cases.
[REQUIRED]: Add tests for all your js files. You are missing the form.js.
[REQUIRED]: In order to follow git flow, kindly make the development branch the default branch.
[OPTIONAL]: It would be great to put your test files in a separate folder.

That's it.

Make the required changes and ask for another review👌👌
Happy coding💻💻

**Review by oloomoses — APPROVED** [review]
Hello @saimaAnis21, 
## Congratulation :muscle: :tada: 
**Well done on the testing :100:  and great job implementing the changes suggested by @NtwaliHeritier :muscle:** 
### This project meets the minimum requirements and therefore Approved :clap: 
**Before merging and closing this pr, Kindly consider working on the optional change suggested below :point_down:**
## Optional
- [ ] Kindly move your test files into a single folder so as to separate them from the main project files.

**I wish best of luck in your next project and see you at the top :rocket:** 

_Slack: Oloo Moses_

**Pull request merged**
